src/main/java/Duke/Tasks/DeleteCommand.java

```diff
@@ -0,0 +1,21 @@
+package Duke.Tasks;
+
+import Duke.Command.Command;
+import Duke.Storage;
+import Duke.TaskList;
+import Duke.Ui;
+
+public class DeleteCommand extends Command {
+
+    int index;
```

Comment:
I think the name index could be changed to something more obvious, like taskIndex to avoid confusion.